src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -300,37 +298,37 @@ public enum EvalMetricType
             SortOrder = 2, ShortName = "nl", NullName = "<Auto>")]
         [TGUI(Description = "The maximum number of leaves per tree", SuggestedSweeps = "2-128;log;inc:4")]
         [TlcModule.SweepableLongParamAttribute("NumLeaves", 2, 128, isLogScale: true, stepSize: 4)]
-        public int? NumLeaves;
+        public int? NumberOfLeaves;
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Minimum number of instances needed in a child.",
             SortOrder = 2, ShortName = "mil", NullName = "<Auto>")]
         [TGUI(Label = "Min Documents In Leaves", SuggestedSweeps = "1,10,20,50 ")]
         [TlcModule.SweepableDiscreteParamAttribute("MinDataPerLeaf", new object[] { 1, 10, 20, 50 })]
-        public int? MinDataPerLeaf;
+        public int? MinimumDataPerLeaf;
 
-        [Argument(ArgumentType.AtMostOnce, HelpText = "Max number of bucket bin for features.", ShortName = "mb")]
-        public int MaxBin = 255;
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum number of bucket bin for features.", ShortName = "mb")]
+        public int MaximumBin = 255;
 
         [Argument(ArgumentType.Multiple, HelpText = "Which booster to use, can be gbtree, gblinear or dart. gbtree and dart use tree based model while gblinear uses linear function.", SortOrder = 3)]
         public ISupportBoosterParameterFactory Booster = new TreeBooster.Options();
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Verbose", ShortName = "v")]
-        public bool VerboseEval = false;
+        public bool Verbose = false;
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Printing running messages.")]
         public bool Silent = true;
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Number of parallel threads used to run LightGBM.", ShortName = "nt")]
-        public int? NThread;
+        public int? NumberOfThreads;
 
         [Argument(ArgumentType.AtMostOnce,
             HelpText = "Evaluation metrics.",
             ShortName = "em")]
-        public EvalMetricType EvalMetric = EvalMetricType.DefaultMetric;
+        public EvalMetricType EvaluationMetric = EvalMetricType.DefaultMetric;
 
         [Argument(ArgumentType.AtMostOnce, HelpText = "Use softmax loss for the multi classification.")]
         [TlcModule.SweepableDiscreteParam("UseSoftmax", new object[] { true, false })]
-        public bool? UseSoftmax;
+        public bool? UseSoftMaximum;
```

Comment:
>UseSoftMaximum [](start = 21, length = 14)

UseSoftmax

Reply:
Need to resolve this...

---
In reply to: [261475871](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261475871) [](ancestors = 261475871)